src/UserAction/JsCallbackExecutor.php

```diff
@@ -62,9 +62,9 @@ public function executeModelAction(array $args = [])
     {
         $this->set(function ($j) {
             // may be id is passed as 'id' or model->id_field within $post args.
-            $id = $_POST['c0'] ?? $_POST['id'] ?? $_POST[$this->action->getOwner()->id_field] ?? null;
-            if ($id && $this->action->appliesTo === Model\UserAction::APPLIES_TO_SINGLE_RECORD) {
-                $this->action->getOwner()->tryLoad($id);
+            $id = $_POST['c0'] ?? $_POST['id'] ?? $_POST[$this->action->getModel()->id_field] ?? null;
+            if ($id && $this->action->appliesTo === Model\UserAction::APPLIES_TO_SINGLE_RECORD && !$this->action->getModel()->isEntity()) {
```

Comment:
`!$this->action->getModel()->isEntity()` will be of course always true ;-)

Reply:
True, I simply forgot to remove it. :-)